<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerShutdownClass</class>
 <widget class="QMainWindow" name="TimerShutdownClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimerShutdown</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon1.ico</normaloff>icon1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="bottomMargin">
             <number>80</number>
            </property>
            <item>
             <widget class="QSpinBox" name="spinBoxHour">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <family>JetBrains Mono</family>
                <pointsize>15</pointsize>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxMinute">
              <property name="font">
               <font>
                <family>JetBrains Mono</family>
                <pointsize>15</pointsize>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxSecond">
              <property name="font">
               <font>
                <family>JetBrains Mono</family>
                <pointsize>15</pointsize>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="startButton">
            <property name="text">
             <string>开始</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="actionBuild_By_thy"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBuild_By_thy">
   <property name="text">
    <string>Build By thy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TimerShutdown.qrc"/>
 </resources>
 <connections/>
</ui>
